js/index.js

```diff
@@ -1,8 +1,160 @@
+    description:
+      "Mijn ambitie is om een goede senior Front-end developer te worden die het team CoDE goed kan begeleiden en ondersteunen op alle fronten ",
+    symbol: "fas fa-user-cog",
+  },
+  {
+    id: 6,
+    title: "Vrije tijd",
+    description:
+      "De meeste vrije tijd breng ik met mijn lieve dochter Lenthè door. Uiteraard is ontwikkellen een een grote hobby daarnaast heb ik nog een paard en een huis van 100 jaar oud waar het nodige aan moet gebeuren. Daar gaat de rest van de tijd inzitten",
+    symbol: "fas fa-battery-three-quarters",
+  },
+  {
+    id: 7,
+    title: "Interest",
+    description:
+      "Mijn interesses liggen bij  HTML, CSS, JS, TS, React en Styled components daar wil ik mij eerst om focussen en daarna kijk ik verder. ",
+    symbol: "fas fa-glasses",
+  },
+  {
+    id: 8,
+    title: "Signature",
+    description:
+      "Wat absoluut kenmerkend aan mij is, is dat ik altijd een goede sfeer in het team probeer te behouden, mensen samen breng en iedereen probeer te helpen waar ik kan.",
+    symbol: "fas fa-signature",
+  },
+  {
+    id: 9,
+    title: "Specialties",
+    description:
+      "Ja dat vind ik altijd een beetje lastig om over mijzelf te zeggen. Mijn dochter vind mij de liefste Mama, dat is ook mijn grootste specialiteit. Mensen blijven graag eten als ik kook. Ben goed in het organiseren van dingen. ",
+    symbol: "fas fa-grin-stars",
+  },
+];
 
+const textDiv = document.querySelector(".karakter_text_wrapper");
+const characters = CharacterJson;
 
 function showKarakter() {
-  var state = document.getElementsByClassName(
-    "karakter__revealer");
-    var aNode = state[0];
-    aNode.style.visibility = aNode.style.visibility === "hidden" ? "visible" : "hidden";
+  let state = document.getElementsByClassName("karakter__revealer");
+  let aNode = state[0];
+  aNode.style.display = aNode.style.display === "none" ? "block" : "none";
 }
+function showKnowledgeBook() {
+  let state = document.getElementsByClassName("knowledgeBook__revealer");
+  let aNode = state[0];
+  aNode.style.display = aNode.style.display === "none" ? "block" : "none";
+}
+
+const iconKarakter = (characters) => {
+  const characterDivID = document.querySelector("#karakter__icon_wrapper");
+  characters.forEach((character) => {
+    const characterDiv = document.createElement("div");
+    const characterButton = document.createElement("button");
+    const characterIcon = document.createElement("i");
+    const characterTitle = document.createElement("h1");
+    const characterDiscription = document.createElement("p");
+
+    characterButton.classList.add("characterBtn");
+    // characterButton.addEventListener(
+    //   "click",
+    //   (event) => showMore(character.description, textDiv) // ik kan hier het carajter id meegeven
+
+    characterButton.addEventListener("click", (event) =>
+      main(character.description, textDiv, character.id)
+    );
+    characterIcon.setAttribute("class", `${character.symbol}`);
```

Comment:
Het is meer correct om een class met behulp van `classList` te setten zoals je op regel 90 ook gedaan hebt.

Reply:
Besproken